Packs/Legacy/Scripts/SetGridField/SetGridField.py

```diff
@@ -0,0 +1,200 @@
+        # Handle entry context list option
+        entry_context_data = [filter_the_dict(item, keys, len(columns)) for item in entry_context_data]
+        table = pd.DataFrame(entry_context_data)
+        table.rename(columns=dict(zip(table.columns, columns)), inplace=True)
+    elif len(entry_context_data) == 1 and isinstance(entry_context_data[0], dict):
+        # Handle entry context key, vlaue option
+        entry_context_data = filter_the_dict(entry_context_data[0], keys).items()
+        table = pd.DataFrame(entry_context_data, columns=columns[:2])
+    else:
+        table = []
+
+    return table
+
+
+def build_grid_command(grid_id: str, context_path: str, keys: List[str], columns: List[str], overwrite: bool, sort_by: str) \
+        -> List[Dict[Any, Any]]:
+    """ Build Grid in one of the 2 options:
+            1. Context_path contain list, e.g. [{'a': 1, 'b': 2}, {'a': 1, 'b': 2}]
+            2. Context_path contain dict (key value pairs), e.g. {'a': 1, 'b': 2}
+
+        Warnings:
+            1. The automation can't validate that the columns name correct and its mandatory.
+            2. The automation knows how to handle only list or dict and doesn't know hoe to handle.
+
+        Args:
+            grid_id: Grid ID to modify.
+            context_path: Entry context path to collect the values from.
+            keys: Keys to be included in the table, If specified "*" will retrieve all availble keys.
+            columns: Name of the columns in the must be equal.
+            overwrite: True if to overwrite existing data else False.
+            sort_by: Name of the column to sort by.
+
+        Returns:
+            list: Table representation for the Grid.
+    """
+    # Get old Data
+    old_table = get_current_table(grid_id=grid_id)
+    # Normalize columns to match connected words.
+    columns = [normalized_string(phrase) for phrase in columns]
+    # Create new Table from the given context path.
+    new_table: pd.DataFrame = build_grid(context_path=context_path,
+                                         keys=keys,
+                                         columns=columns)
+    # Merge tabels if not specified to overwrite.
+    if not overwrite:
+        new_table = pd.concat([new_table, old_table])
+    # Sory by column name if specified
+    if sort_by and sort_by in new_table.columns:
+        new_table.sort_values(by=sort_by)
+
+    return new_table.to_dict(orient='records')
+
+
+def main():
+    try:
+        # Normalize grid id from any form to connected lower words, e.g. my_word/myWord -> myword
+        grid_id = normalized_string(demisto.getArg('grid_id'))
+        # Build updated table
+        table = build_grid_command(grid_id=grid_id,
+                                   context_path=demisto.getArg('context_path'),
+                                   keys=argToList(demisto.getArg('keys')),
+                                   overwrite=False if demisto.getArg('overwrite') == 'false' else True,
```

Comment:
invalid input will not override
```suggestion
                                   overwrite=demisto.getArg('overwrite').lower() == 'true',
```